test/core-flags/module-flags.test.js

```diff
@@ -2,8 +2,8 @@
 
 const { run, hyphenToUpperCase } = require('../utils/test-utils');
 const { flagsFromCore } = require('../../packages/webpack-cli/lib/utils/cli-flags');
-//--module-rules-options will be excluded for cli.
-const moduleFlags = flagsFromCore.filter(({ name }) => name.startsWith('module-') && name !== 'module-rules-options');
+
+const moduleFlags = flagsFromCore.filter(({ name }) => name.startsWith('module-'));
```

Comment:
/cc @snitin315 So we still don't have tests on `module-` flags?

Reply:
@evilebottnawi  We have, This file contains exclusively tests for all `module-*` flags, We are extracting all the `module-*` flags here and then looping through them. 
https://github.com/webpack/webpack-cli/blob/4fe80450aa4c29f3603b5cecea7d2a0f2db59658/test/core-flags/module-flags.test.js#L8-L9

Reply:
Thanks for clarify!